src/rosdistro/manifest_provider/git.py

```diff
@@ -104,7 +119,7 @@ def check_remote_tag_exists(url, tag):
     return False
 
 
-def _run_command(cmd, cwd, env=None):
+def _run_command(cmd, cwd='/', env=None):
```

Comment:
I don't think that a default working directory of '/'  is a good idea. Instead please make the default `None` which implies the current working directory.